src/bptl/work_units/xential/views.py

```diff
@@ -0,0 +1,73 @@
+from uuid import UUID
+
+from django.http import HttpResponse, HttpResponseRedirect
+from django.views import View
+
+from rest_framework import status, views
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from bptl.work_units.xential.models import XentialTicket
+from bptl.work_units.xential.serializers import CallbackDataSerializer
+
+from ...tasks.base import check_variable
+from .client import DRC_ALIAS, XENTIAL_ALIAS, get_client
+from .handlers import on_document_created
+from .utils import get_xential_base_url, parse_xml
+
+
+class DocumentCreationCallbackView(views.APIView):
+    permission_classes = []
+
+    def post(self, request: Request) -> Response:
+        # The callback sends the base64 encoded document and the BPTL ticket ID as XML.
+        callback_data = parse_xml(request.data)
```

Comment:
Might be worth to consider https://jpadilla.github.io/django-rest-framework-xml/

Reply:
Added this in https://github.com/GemeenteUtrecht/bptl/pull/128